eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ArchiveWSServer.java

```diff
@@ -127,20 +119,20 @@ public QueryResponse getStudiesStats(@RequestParam(name = "species", required =
                                          @RequestParam(name = "structural", defaultValue = "false") boolean structural) {
         initializeQuery();
         if (species != null && !species.isEmpty()) {
-            queryOptions.put("species", species);
+            getQueryOptions().put("species", species);
```

Comment:
There are changes to this in the PR https://github.com/EBIvariation/eva-ws/pull/52. No local variable is created, but query utils is used directly.